tests/foreman/ui/test_ansible.py

```diff
@@ -875,3 +876,62 @@ def test_positive_schedule_recurring_hostgroup_job(self):
 
         :expectedresults: Scheduled Job appears in the Job Invocation list at the appointed time
         """
+
+    @pytest.mark.no_containers
+    @pytest.mark.rhel_ver_list([settings.content_host.default_rhel_version])
+    @pytest.mark.parametrize('setting_update', ['ansible_verbosity'], indirect=True)
+    def test_positive_ansible_job_with_verbose_stdout(
+        self,
+        target_sat,
+        module_org,
+        module_location,
+        module_ak_with_cv,
+        setting_update,
+        registered_hosts,
+    ):
+        """Verify ansible_verbosity setting and dynflow console output for expected hosts
+
+        :id: 06e0a95c-530b-11ef-a28c-36dcd2c0c415
+
+        :steps:
+            1. Select two or more hosts
+            2. Run ansible roles on those hosts
+            3. Check the Ansible job execution logs
+
+        :expectedresults: Ansible job console stdout should mention the verbose logs for expected host instead of the all host present in the inventory.
+
+        :BZ: 1912941
+
+        :Verifies: SAT-12267
+
+        :customerscenario: true
+        """
+        if not is_open('SAT-27757'):
+            setting_update.value = '3'
+            setting_update.update({'value'})
```

Comment:
How about reverting the setting back to its default value?
```suggestion
        if not is_open('SAT-27757'):
            setting_update.value = '3'
            setting_update.update({'value'})
            @request.addfinalizer
             def _finalize():
                 setting_update.value = '0'
                 setting_update.update({'value'})
```

Reply:
We can do that, but I believe our goal is to check the output with only '3'. Is it necessary to set it as the default again? What do you think?

Reply:
I have added a finalizer that will set the default value in the settings.